curriculum/challenges/english/25-front-end-development/lecture-understanding-core-javascript-fundamentals/6732c6ab21031b60d2b0c999.md

```diff
@@ -86,35 +86,35 @@ Remember the prefix we typically use for getter functions.
 
 ## --text--
 
-Which of these is the best name for a constant representing the maximum number of login attempts?
+Which of the following should you do if you’re working in an established codebase and / or with a team?
 
 ## --answers--
 
-`max_attempts`
+Use your text editor’s find-and-replace feature to update any function or variable names you come across that could be improved, even it's not related to the task you're working on.
 
 ### --feedback--
 
-Recall the convention for naming constants in JavaScript.
+Remember that the code you write should be readable and maintainable for everyone.
 
 ---
 
-`MAX_LOGIN_ATTEMPTS`
+Stick to the naming conventions for variables and functions that have already been established.
 
 ---
 
-`loginMaxAttempts`
+Come up with your own naming conventions for any new code you write.
 
 ### --feedback--
 
-Recall the convention for naming constants in JavaScript.
+Remember that the code you write should be readable and maintainable for everyone.
 
 ---
 
-`ATTEMPTS_MAX`
+Strictly use the naming conventions covered in this lecture.
 
 ### --feedback--
 
-Recall the convention for naming constants in JavaScript.
+Remember that the code you write should be readable and maintainable for everyone.
```

Comment:
```suggestion
Remember that this lecture covers some naming conventions, each codebase could have its own established conventions.
```

Reply:
Thanks for this suggestion, @ilenia-magoni. This sounds good to me. I'm just going to tweak it slightly to "Remember that this lecture covers some naming conventions, and each codebase could have its own established conventions."